### DISCUSSION TITLE
Dripsy UI ?

### DISCUSSION THREAD

**Discussion by fortezhuo** [nandorojo/dripsy#144] [Ideas]
I really2 love your work about dripsy and moti. But unfortunately, not everyone (include me) have great design skill to create component. And just wonder, if Dripsy have any roadmap to create more component as new UI framework like Chakra UI ?

**Comment by nandorojo** [comment]
Moved this to a discussion.

It's certainly something I've thought about. But it's a lot of work to make something truly good, cross-platform, and (perhaps) free. I would say it depends on the level of general interest and buy-in from people. But I do like the idea of, at the very least, a design system. Perhaps #140 is also relevant (see my comments at the bottom)
